tests/variational/test_callbacks.py

```diff
@@ -52,3 +52,33 @@ def test_tracker_callback():
     tracker = pm.callbacks.Tracker(bad=lambda t: t)  # bad signature
     with pytest.raises(TypeError):
         tracker(None, None, 1)
+
+
+def test_reducelronplateau_callback():
+    optimizer = pm.adam(learning_rate=0.1)
```

Comment:
Use a pytest.mark.parameterize here to test all the optimizers and make sure there's no unexpected behaviors between them